TUMFARSynchrony/experimental-hub
# Add groupFilters to Frontend with dynamic filters

This PR adds groupFilters as dynamic filters to the Frontend.
There were also some refactoring changes in the backend.

A new Data Type was introduced: https://github.com/TUMFARSynchrony/experimental-hub/wiki/Data-Types#filterconfig
and a new API call: https://github.com/TUMFARSynchrony/experimental-hub/wiki/Backend-API#get_filters_config

-------------------------------------------------------------------
Testing:
- in backend/group_filters/template/template_group_filter.py: change filter_type function from `NONE` to `SESSION`
- create a new experiment
- add one Participant and add a groupFilter -> should be also listed
- save experiment
- look under participants, if group_filter is listed
